packages/patternfly-react/src/components/TypeAheadSelect/TypeAheadSelect.js

```diff
@@ -1 +0,0 @@
-export { Typeahead as TypeAheadSelect } from 'react-bootstrap-typeahead';
```

Comment:
i'm a tad concerned that this could break any applications that are importing directly from this file (though using index would be better practice).

Reply:
@jeff-phillips-18, I guess you are right,
I will keep this file and than export it to index :)